modules/apps/address/address-impl/src/main/resources/com/liferay/address/dependencies/regions/GB.json

```diff
@@ -548,8 +556,8 @@
 		"regionCode": "NYK"
 	},
 	{
-		"name": "Northamptonshire",
-		"regionCode": "NTH"
```

Comment:
What about the already existing addresses that are using this regioneCode?

Reply:
It is separated two unary regions [West Northamptonshire](https://en.wikipedia.org/wiki/West_Northamptonshire)
[North Northamptonshire](https://en.wikipedia.org/wiki/North_Northamptonshire)